### PR TITLE
new privilege docs

### PR DESCRIPTION
**Pull request by sravfeyn**

### CODE REVIEW

**Comment by sravfeyn** [comment]
I keep forgetting what needs to be done to create a new privilege and figuring out from just the code takes more time than necessary. So I am adding a simple docs for that.

**Comment by sravfeyn** [comment]
@esoergel

File: corehq/apps/accounting/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+### Adding a new Privilege
+
```
Comment (sravfeyn) [inline corehq/apps/accounting/README.md]: With our Django upgrade almost ready, this no longer might be True at all. Please feel free to update this, if any of this is wrong.

Comment (snopoke) [inline corehq/apps/accounting/README.md]: Updated and added utility function

Comment (sravfeyn) [inline corehq/apps/accounting/README.md]: Awesome, thanks simon!

**Pull request merged**